src/libraries/System.Private.CoreLib/src/System/Reflection/MdSigCallingConvention.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Reflection
+{
+    /// <summary>
+    /// Represents the first byte of a method signature.
+    /// Calling conventions have been extended into modopts for Unmanaged.
+    /// </summary>
```

Comment:
First byte of a method signature if the method signature is in the ECMA-335 file format. Native AOT doesn't use this file format.

Could we have a separate enumeration just with the calling conventions? Or reuse S.R.InteropServices.CallingConvention?

Reply:
I'll create a new enumeration. The one in `S.R.InteropServices.CallingConvention` is different; for `[DllImport]`.